MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -50,8 +50,8 @@ class CarPlayMapViewController: UIViewController, MLNMapViewDelegate {
         super.viewDidLoad()
 
         self.styleManager = StyleManager(self)
-        self.styleManager.styles = [DayStyle(), NightStyle()]
-        
+        self.styleManager.styles = [DayStyle(demoStyle: ()), NightStyle(demoStyle: ())]
```

Comment:
The demo styles are now being used in some places that were previously defaulting to mapbox styles. This is probably wrong, but (I think) no worse than things being completely broken before.

Probably these should somehow be surfaced or inferred as well. I somewhat arbitrarily decided this was out of scope for this PR, but I could follow up in this PR or another depending on how people are feeling about it.

Reply:
I personally don't mind your approach here 👍